cmd/db/db.go

```diff
@@ -22,23 +23,48 @@ type DB struct {
 	db *sql.DB
 }
 
-func NewDatabase(ctx context.Context, user apiv1alpha1.SystemUser, pass, host string, port int32) (*DB, error) {
+type DBParams struct {
+	User apiv1alpha1.SystemUser
+	Pass string
+	Host string
+	Port int32
+
+	ReadTimeout uint32 // in seconds
```

Comment:
I think we could incorporate the seconds property to the name of the variable e.g. `ReadTimeoutSeconds`. This will make it more clear throughout the codebase without having to rely on the comment to clarify it.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/838/commits/6911539a62bff189460a572cefa054cbfb1b03ce